firestore/functions/src/index.ts

```diff
@@ -0,0 +1,111 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import * as util from 'util';
+import { DocumentSnapshot, DocumentData } from '@google-cloud/firestore';
+// // Start writing Firebase Functions
+// // https://firebase.google.com/docs/functions/typescript
+//
+
+"use strict";
+
+// Uncomment these lines to run our Cloud Functions in the emulator
+
+// const sa = require( './super_secret/new-test-projects-a746c-firebase-adminsdk-ttdbm-3227776c2b.json');
+ 
+// admin.initializeApp({
+//   credential: admin.credential.cert(sa),
+//   databaseURL: "http://localhost:8080/"
+// });
+ 
+admin.initializeApp();
```

Comment:
*Personal preference*

I like to return the app from `initializeApp()` instead of relying on a global app.

```ts
const app = admin.initializeApp();
const firestore = app.firestore();
```

Reply:
Done